core/uvisor-config.h

```diff
@@ -27,7 +27,13 @@
  * link failure, so it can be used to track down a rough estimation of the
  * uVisor flash usage upper bound.
  */
-#define UVISOR_FLASH_LENGTH_MAX 0x9C00
+#if NDEBUG
+/* Release builds can be up to this big. */
```

Comment:
How does this work when we preprocess the linker script? The options for the preprocessing step are the following:

```make
LINKER_CONFIG:=\
    -D$(CONFIGURATION) \
    -I$(PLATFORM_DIR)/$(PLATFORM)/inc \
    -include $(CORE_DIR)/uvisor-config.h
```

But `NDEBUG` is not set there.